package.json

```diff
@@ -10,8 +10,8 @@
     "build:netlify": "vite build -c vite.config.netlify.ts"
   },
   "dependencies": {
-    "@spryker-oryx/themes": "1.0.0-rc.1",
-    "@spryker-oryx/presets": "1.0.0-rc.1"
+    "@spryker-oryx/themes": "rc",
+    "@spryker-oryx/presets": "rc"
```

Comment:
think it should be `latest`, but we need to have that on npm as well

Reply:
We will return to latest once we release 1.0.0.